src/site/stages/build/plugins/configure-assets.js

```diff
@@ -14,8 +14,10 @@ function configureAssets(smith, buildOptions) {
 
   if (buildOptions.watch) {
     const watchPaths = {
-      [`${buildOptions.contentRoot}/**/*`]: '**/*.{md,html}',
```

Comment:
I removed this because it seems like a waste of memory to watch the whole content repo when only the pages are likely to be changed.